lesson_09/test_homeworks09.py

```diff
@@ -1,11 +1,56 @@
-""" Оберіть від 3 до 5 різних домашніх завдань
-- перетворюєте їх у функції (якщо це потрібно)
-- створіть в папці файл homeworks.py куди вставте ваші функції з дз
-- та покрийте їх не менш ніж 10 тестами (це загальна к-сть на все ДЗ).
--- імпорт та самі тести помістіть в окремому файлі - test_homeworks09.py
-Код закомітьте в гіт, надайте посилання.
-На оцінку впливає як якість тестів так і розмір тестового покриття.
-Мінімум на 10 балів - 1 правильно задизайнений позитивний тест на функцію.
-"""
+import unittest
+import math
 import homeworks
+import pytest
```

Comment:
```suggestion
import pytest
```
а навіщо пайтест, якщо тести на юніттест?

Reply:
Зайве мабуть буде в рамках цього дз, просто вирішив поклацати ще і пайтест в рамках цієї домашки